src/main/java/com/google/step/finscholar/data/College.java

```diff
@@ -0,0 +1,163 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.finscholar;
+
+import java.util.List;
+import java.util.UUID;
+
+/** An object representing a College. */
+public class College {
+
+  private String schoolName;
+  private UUID collegeUuid;
+  private String institutionType;
+  private double acceptanceRate;
+  private double averageACTScore;
+  private List<UUID> usersUUIDList;
+  private int totalCostAttendance;
+  private int netCostForFirstQuintile; // $0-$30,000.
+  private int netCostForSecondQuintile; // $30,001-$48,000.
+  private int netCostForThirdQuintile;  // $48,001-$75,000.
+  private int netCostForFourthQuintile; // $75,001-$110,000.
+  private int netCostForFifthQuintile; //$110,000+.
+  private int cumulativeMedianDebt;
+
+  // Constructor.
+
+  public College(String schoolName) {
+    this.schoolName = schoolName;
+    this.collegeUuid = generateUUID();
+  }
+
+  // Private helper methods.
+
+  private UUID generateUUID() {
+    return UUID.randomUUID();
+  }
+
+  // Getter methods.
+
+  /**
+   * @return - Name of the college.
```

Comment:
You may be able to make these a single line.
`/** @return - Name of the college. */`

Reply:
Good idea!